File: src/main/java/itmo/is/project/repository/module/storage/StorageModuleBlueprintRepository.java
package itmo.is.project.repository.module.storage;

import itmo.is.project.model.module.storage.StorageModuleBlueprint;
import itmo.is.project.repository.module.ModuleBlueprintRepository;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface StorageModuleBlueprintRepository extends ModuleBlueprintRepository<StorageModuleBlueprint> {
}

File: src/main/kotlin/itmo/isproject/repository/module/storage/StorageModuleBlueprintRepository.kt
package itmo.isproject.repository.module.storage

import itmo.isproject.model.module.storage.StorageModuleBlueprint
import itmo.isproject.repository.module.ModuleBlueprintRepository
import org.springframework.stereotype.Repository

@Repository
interface StorageModuleBlueprintRepository : ModuleBlueprintRepository<StorageModuleBlueprint>
